### PR TITLE
Improved build process on Travis CI

### PR DESCRIPTION
**Pull request by sergeyklay**
- Improved build process on Travis CI
- Fixed code style
- Setting up the PHP_CodeSniffer
- Add PHP 5.6 support
- Moved all tests to the separated namespace

---

Refs: https://github.com/jonahgeorge/jaeger-client-php/pull/46

### CODE REVIEW

**Comment by sergeyklay** [comment]
This PR covers #46 and in general related to code style fixes to satisfy PSR-2 Coding Standard. Also I've improved build process on Travis CS. 

/cc @jonahgeorge

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: '5.6'
+    - php: nightly
+
+cache:
+  apt: true
+  timeout: 604800
```
Comment (jonahgeorge) [inline .travis.yml]: I'm under the impression that setting the timeout here likely isn't necessary. The build is fast enough and I'd prefer to not have to debug issues around a stale cache.

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+  directories:
+    - $HOME/.composer/cache
+
+before_install:
+  # https://stackoverflow.com/a/18029286/1661465
+  - if [ -n "$GH_TOKEN" ]; then composer config github-oauth.github.com $GH_TOKEN; fi;
```
Comment (jonahgeorge) [inline .travis.yml]: Would you clarify why this is being set?

**Comment by sergeyklay** [comment]
@jonahgeorge I made all the required changes :)

**Pull request merged**